doc/programming/examples/_parse-if-block-with-else.spicy

```diff
@@ -0,0 +1,15 @@
+# Automatically generated; edit in Sphinx source code, not here.
+module Test;
+
+public type Foo = unit {
+    a: int8;
+
+    if ( self.a == 1 ) {
+        b: int8;
+    }
+    else {
```

Comment:
With my `spicy-format` hat on I feel we should format this on a single line `} else {`, if only since it makes the absence of a `;` after the `if` block less surprising.

Reply:
Not sure I like that ...